packages/ui/src/form/Fieldset/Fieldset.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { render, screen, within } from '@bigcommerce/checkout/test-utils';
+
+import Fieldset from './Fieldset';
+
+describe('Fieldset', () => {
+    it('renders component with test ID, legend and children', () => {
+        const { container } = render(
+            <Fieldset legend={<legend>Hello world</legend>} testId="test">
+                <input type="text" />
+            </Fieldset>,
+        );
+
+        expect(screen.getByTestId('test')).toBeInTheDocument();
+        // eslint-disable-next-line testing-library/no-container
+        expect(container.querySelector('fieldset')).toBeInTheDocument();
+        // eslint-disable-next-line testing-library/no-container
+        expect(container.querySelector('legend')).toBeInTheDocument();
+        // eslint-disable-next-line testing-library/no-container
```

Comment:
Next time, you might consider using `/* eslint-disable testing-library/no-container */` to disable this rule for the whole file.